### PR TITLE
Allow character parsing

### PR DESCRIPTION
**Pull request by joshinat0r**
- Updated packages
- added support for `https://www.warcraftlogs.com/character/{region}/{realm}/{name}` and `https://www.warcraftlogs.com/character/id/{id}` URL (by parsing the DOM content)
- minor layout tweaks

Not quite sure who is handling the updates for the webstores.

image
image

### CODE REVIEW

File: src/scripts/popup.js [hunk 1]
```diff
@@ -1,21 +1,40 @@
 import ext from "./utils/ext";
+import PARSE_TYPES from './PARSE_TYPES';
 
-var getUrl = (code) => {
-  return `https://wowanalyzer.com/report/${code}`;
+var getUrl = (code, type) => {
+  console.log("renderBookmark");
```
Comment (MartijnHols) [inline src/scripts/popup.js]: Please remove console statements from production code

**Review by MartijnHols — DISMISSED** [review]
Thanks! I must admit I have no clue how to deploy this, and there's no CD so it will have to be manually done afaik. @Yuyz0112 (Aryu on Discord) made the original version, he might be able to shed more light on the deployment process.

**Comment by joshinat0r** [comment]
Have no idea either, running `npm run dist` make two zips.
I guess they have to get uploaded somewhere.

And it does look like Firefox has no addon right now, could only find it in the chrome webstore

**Review by MartijnHols — APPROVED** [review]

**Comment by Yuyz0112** [comment]
Sorry for the late.
Currently the extension was published by my google account, but I think it would be better to be published by @MartijnHols .

I will investigate the workflow of migrating or republishing the extension this weekend. Maybe we can discuss about it later in the discord.

**Pull request merged**